### PR TITLE
Translation/updates po 01

### PR DESCRIPTION
**Pull request by cmgustavo**
* Fixes translation of date
* Adds strings
* Updates .pot and .po files

### CODE REVIEW

**Comment by dabura667** [comment]
@cmgustavo did you just overwrite the Copay template.pot with the Bitpay template.pot on Crowdin?

This means that until you merge all text updates from Bitpay to Copay you can no longer build the Copay translations... is that ok?

**Comment by dabura667** [comment]
... and this is why usually people separate apps into separate repositories... :cry:

File: www/views/tab-settings.html [hunk 1]
```diff
@@ -33,31 +33,31 @@
         <i class="icon big-icon-svg">
           <img src="img/icon-language.svg" class="bg"/>
         </i>
-        <span translate>Send Feedback</span>
+        <span>{{'Send Feedback' | translate}}</span>
         <i class="icon bp-arrow-right"></i>
       </a>
       <a ng-if="isCordova" class="item item-icon-left item-icon-right" ui-sref="tabs.shareApp({score: 4, skipped: true, fromSettings: true})">
         <i class="icon big-icon-svg">
           <img src="img/icon-send-feedback.svg" class="bg"/>
```
Comment (gabrielbazan7) [inline www/views/tab-settings.html]: here `img/icon-heart.svg`

File: www/views/tab-settings.html [hunk 2]
```diff
@@ -139,7 +139,7 @@
         <i class="icon big-icon-svg">
           <img src="img/icon-heart.svg" class="bg"/>
         </i>
-        <span translate>Tell a friend</span>
+        <span>{{'Tell a friend' | translate}}</span>
```
Comment (gabrielbazan7) [inline www/views/tab-settings.html]: All this `<a>` needs to be removed

**Pull request merged**